moveit_ros/moveit_servo/include/moveit_servo/servo_calcs.h

```diff
@@ -92,6 +92,17 @@ class ServoCalcs
    */
   void start();
 
+  /***
+   * Validate the parameter values
+   */
+  void validateParams();
```

Comment:
I missed why we needed this function when `generate_parameter_library` already has parameter validators.

Might be worth saying in the comment that this validates not the individual parameter, but any additional parameters that are interdependent (like the self vs scene collision threshold, etc.)